healpix/core.py

```diff
@@ -341,7 +389,7 @@ def interpolate_bilinear_lonlat(lon, lat, values, order='ring'):
     if values.ndim != 1:
         raise ValueError("values must be a 1-dimensional array")
 
-    return core_cython.interpolate_bilinear_lonlat(lon, lat, values, order)
+    return _restore_shape(core_cython.interpolate_bilinear_lonlat(lon, lat, values, order), shape=shape)
```

Comment:
This function still contains the TODO and ndim check a few lines up.
It should be removed, no?

Matter of taste: I would separate the Cython call from the reshape, i.e. introduce an extra line here.

Reply:
Sounds good, will fix

Reply:
The TODO is about something else, it is if we want to support e.g. FITS files with vector columns for the actual map value.